client/src/pages/navigation-step-by-step-forward.tsx

```diff
@@ -59,23 +60,36 @@ export default function Navigation(){
     let mapElements: MapDrawElement[] = [];
     let submapImage: Submap | undefined;
     let centroidCrop: CentroidScale;
+    let rotateAngle = 0;
+
     if (navSteps !== undefined && navSteps.length > 0) {
         currentStep = navSteps[currentStepIndex];
+        submapImage = submap.getSubmapImage(currentStep.nodes[0].submapId);
         let prevDot = {} as Dot;
+
         currentStep.nodes.forEach((node, index) => {
             const dot = {x: node.xCoordinate, y: node.yCoordinate} as Dot;
             const mapDot = new MapDot(dot, "#41C7F7", 0.5, 1);
             mapElements.push(mapDot);
+
             if(index > 0){
                 const line = {dot1: prevDot, dot2: dot} as Line;
                 const mapLine = new MapPathLine(line, "#41C7F7", 0.1);
                 mapElements.push(mapLine);
+
+                if(index == 1){
+                    const referenceLine : Line = {dot1: ORIGIN_POINT, dot2: {x: 0, y: -1}};
```

Comment:
I don't know why having an "origin point" constant helps here. Furthermore, that constant makes me think of why this line must be at the origin point and, even more, what is the origin point and why it's so important.

You wanted to just create a vector line "looking to" negative `y` direction? If so, I think just using `{x: 0, y: 0}` instead of the constant if simpler and easier to read.